api/src/main/java/org/openmrs/module/openconceptlab/Subscription.java

```diff
@@ -86,11 +94,11 @@ public boolean isManual() {
 		return (days == null && hours == null && minutes == null) || (days == 0 && hours == 0 && minutes == 0);
 	}
 
-	public boolean isSubscribedToSnapshot() {
+	public Boolean isSubscribedToSnapshot() {
```

Comment:
When I change it to boolean I'm getting some validation exception, I am not able to post Subscription at all because of this.